packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -12,7 +12,8 @@ export interface FieldDefinition {
   label?: string;
   uuid: string;
   placeholder?: string;
```

Comment:
May as well remove placeholder here too? Also, these should be removed from the _actual config schema_ and not just the types.